modules/lightning_features/lightning_media/tests/src/FunctionalJavascript/MediaReferenceFieldTest.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace Drupal\Tests\lightning_media\FunctionalJavascript;
+
+use Drupal\FunctionalJavascriptTests\JavascriptTestBase;
+
+/**
+ * @group lightning
+ * @group lightning_media
+ */
+class MediaReferenceFieldTest extends JavascriptTestBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  protected $profile = 'lightning';
+
+  /**
+   * {@inheritdoc}
+   */
+  protected $strictConfigSchema = FALSE;
```

Comment:
Why this is false? Which config schema/metadata is missing? Could we add a todo to remove this after adding the schema/metadata if defining metadata is out of the scope of this issue.